### PR TITLE
tidb: add description for var `tidb_ddl_reorg_max_write_speed`

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

add description for var `tidb_ddl_reorg_max_write_speed`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/57145

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1516,6 +1516,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 1125899906842624]`（1 PiB）
+- 添加索引加速功能开启（由变量 `tidb_ddl_enable_fast_reorg` 控制）时，这个变量用于限制向每个 TiKV 节点写入的带宽大小，单位为字节。 默认值 `0` 表示不限制写入带宽。可以通过 `'1GiB'`、`256MiB` 等方式设置该变量。
```
Comment (tangenta) [inline system-variables.md:1527]:
```suggestion
- 添加索引加速功能开启（由变量 `tidb_ddl_enable_fast_reorg` 控制）时，这个变量用于限制向每个 TiKV 节点写入的带宽大小，单位为字节每秒。 默认值 `0` 表示不限制写入带宽。可以通过 `'1GiB'`、`256MiB` 等方式设置该变量。
```

File: system-variables.md [hunk 2]
```diff
@@ -1516,6 +1516,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 可选值：`PRIORITY_LOW`、`PRIORITY_NORMAL`、`PRIORITY_HIGH`
 - 这个变量用来设置 `ADD INDEX` 操作 `re-organize` 阶段的执行优先级，可设置为 `PRIORITY_LOW`/`PRIORITY_NORMAL`/`PRIORITY_HIGH`。
 
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
```
Comment (tangenta) [inline system-variables.md:1525]: 默认值可能要等测试结果出来再看看？cc @Frank945946

**Review by tangenta — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1516,6 +1516,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+- 范围：`[0, 1125899906842624]`（1 PiB）
+- 添加索引加速功能开启（由变量 `tidb_ddl_enable_fast_reorg` 控制）时，这个变量用于限制向每个 TiKV 节点写入的带宽大小，单位为字节每秒。 默认值 `0` 表示不限制写入带宽。可以通过 `'1GiB'`、`256MiB` 等方式设置该变量。
```
Comment (qiancai) [inline system-variables.md:1527]:
```suggestion
- 该变量仅在添加索引加速功能开启（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）时生效，用于限制每个 TiKV 节点写入的带宽。 默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`256MiB` 等格式设置该变量。
```

File: system-variables.md [hunk 4]
```diff
@@ -1516,6 +1516,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 这个变量用来设置 `ADD INDEX` 操作 `re-organize` 阶段的执行优先级，可设置为 `PRIORITY_LOW`/`PRIORITY_NORMAL`/`PRIORITY_HIGH`。
 
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 1125899906842624]`（1 PiB）
```
Comment (qiancai) [inline system-variables.md:1526]:
```suggestion
- 范围：`[0, 1125899906842624]`（即最大可设置为 1 PiB）
```

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -1516,6 +1516,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 1125899906842624]`（即最大可设置为 1 PiB）
+- 该变量仅在添加索引加速功能开启（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）时生效，用于限制每个 TiKV 节点写入的带宽。 默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`256MiB` 等格式设置该变量。
```
Comment (hfxsd) [inline system-variables.md:1527]:
```suggestion
- 该变量仅在添加索引加速功能开启（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）时生效，用于限制每个 TiKV 节点写入的带宽。默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`256MiB` 等格式设置该变量。
```

Comment (Oreoxmt) [inline system-variables.md:1527]:
```suggestion
- 这个变量用于限制每个 TiKV 节点写入的带宽，仅在开启添加索引加速功能时生效（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）。默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`256MiB` 等格式设置该变量。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by tangenta — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -1516,6 +1516,17 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 可选值：`PRIORITY_LOW`、`PRIORITY_NORMAL`、`PRIORITY_HIGH`
 - 这个变量用来设置 `ADD INDEX` 操作 `re-organize` 阶段的执行优先级，可设置为 `PRIORITY_LOW`/`PRIORITY_NORMAL`/`PRIORITY_HIGH`。
 
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
```
Comment (tangenta) [inline system-variables.md:1519]: 如果这个 feature 往旧版本 pick，文档里还写 v8.5.0 引入吗

Comment (qiancai) [reply system-variables.md:1519]: 是的，等以后新的 patch 发版了，需要修改这里把 patch 的信息也带上😂

比如
image

File: system-variables.md [hunk 7]
```diff
@@ -1516,6 +1516,17 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 1125899906842624]`（即最大可设置为 1 PiB）
+- 这个变量用于限制每个 TiKV 节点写入的带宽，仅在开启添加索引加速功能时生效（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）。在数据量特别大的情况下，降低加索引时写入 TiKV 节点的带宽可以有效减少对前端负载的影响。
```
Comment (qiancai) [inline system-variables.md:1527]:
```suggestion
- 这个变量用于限制每个 TiKV 节点写入的带宽，仅在开启添加索引加速功能时生效（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）。在数据量特别大的情况下（如数十亿行数据），降低加索引时写入 TiKV 节点的带宽可以有效减少对业务负载的影响。
```

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 8]
```diff
@@ -1516,6 +1516,17 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+### `tidb_ddl_reorg_max_write_speed` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 1125899906842624]`（即最大可设置为 1 PiB）
+- 这个变量用于限制每个 TiKV 节点写入的带宽，仅在开启添加索引加速功能时生效（由变量 [`tidb_ddl_enable_fast_reorg`](#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) 控制）。在数据量特别大的情况下（如数十亿行数据），降低加索引时写入 TiKV 节点的带宽可以有效减少对业务负载的影响。
+- 默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`256MiB` 等格式设置该变量。
```
Comment (qiancai) [inline system-variables.md:1528]:
```suggestion
- 默认值 `0` 表示不限制写入带宽。默认单位为字节每秒，也可以通过 `'1GiB'`、`'256MiB'` 等格式设置该变量。
```

**Pull request merged**